crates/milli/src/update/new/indexer/post_processing.rs

```diff
@@ -131,7 +131,12 @@ fn compute_word_fst(
     }
 }
 
-pub fn recompute_word_fst_from_word_docids_database(index: &Index, wtxn: &mut RwTxn) -> Result<()> {
+pub fn recompute_word_fst_from_word_docids_database(
+    index: &Index,
+    wtxn: &mut RwTxn,
+    progress: &Progress,
+) -> Result<()> {
+    progress.update_progress(PostProcessingWords::WordFst);
```

Comment:
ah yes, by "follow the fst progress", I meant like each step of progress, like how many words do we have to build, and how many words we built already.